shell/components/arc-tools/handle-explorer.js

```diff
@@ -73,6 +73,9 @@ class HandleExplorer extends Xen.Base {
     const result = [];
     if (handles) {
       for (let [handle, tags] of handles) {
+        if (handle.name === null) {
```

Comment:
Since it's a dev tool, it could be worth still sticking it in with just id or whatever it has. Just a thought while reading, I'm not sure what you actually saw that prompted this.

Reply:
There tends to be numerous internal handles (created by runtime processing) that specifically have `name === null` and IMO don't reveal generally useful dev information, so I filtered them out. If it turns out somebody actually needs to see these we can revisit, or add an option.